server/queries/user.js

```diff
@@ -11,4 +11,8 @@ function insertUser(params) {
   });
 }
 
-module.exports = { insertUser };
+function getUserByFieldName(field, value) {
```

Comment:
can we simplify the name of this function to something like:
getUserBy(field, value)
because field is anyway passed in the arguments right?

So it will look better to read i think, like this:
`getUserBy(username, "magesh")`
what do you think?

Reply:
yeah cool